EmailController.java

```diff
@@ -0,0 +1,74 @@
+package com.example.demo;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.MediaType;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.ArrayList;
+import java.util.Date;
+import java.util.stream.Collectors;
+
+import static org.springframework.web.bind.annotation.RequestMethod.DELETE;
+import static org.springframework.web.bind.annotation.RequestMethod.GET;
+import static org.springframework.web.bind.annotation.RequestMethod.POST;
+
+@RestController()
+@RequestMapping("/Email")
+public class EmailController {
+
+    WorkWithJsonFile workWithJsonFile;
+
+    @Autowired
+    public EmailController(WorkWithJsonFile workWithJsonFile) {
+        this.workWithJsonFile = workWithJsonFile;
+    }
+
+    @RequestMapping(value = "/activeEmails", method=GET)
+    private ArrayList<Email> showsActiveEmails(){
+        return (ArrayList<Email>) workWithJsonFile.loadFromJsonEmailsList().stream()
+                                                        .filter(email -> email.getDeliveryDate().after(new Date()) && !email.getSended())
+                                                        .collect(Collectors.toList());
+    }
+
+    @RequestMapping(value = "/newEmail", method=POST,consumes = MediaType.APPLICATION_JSON_VALUE)
+    private void addNew(@RequestBody Email newEmail){
+        workWithJsonFile.saveEmailToJson(newEmail);
+    }
+
+    @DeleteMapping(value = "/delete")
+    private boolean deleteOne(@RequestBody Email newEmail){
```

Comment:
How boolean value be serialized into response? As plain text? As JSON? If the method you design should not return anything, just use void type. If you want to signal operation result(success or failure) it's better to use http status codes. 200(Ok) if email was found and deleted successfully and 404(Not found) when no email represented by identifier